CONTRIBUTING.md

```diff
@@ -131,6 +131,15 @@ Here's an example for `homebrew`:
 brew install diffutils
 ```
 
+### Update test snapshots
+
+If you're contributing to our test suite, make sure you update (and then check!)
+the test snapshots:
+
+```shell
+UPDATE_SNAPSHOTS=1 yarn jest __tests__/test.file.graphql
+```
```

Comment:
````suggestion
```shell
UPDATE_SNAPSHOTS=1 yarn test
```

If you're iterating you may want to test just a single file; to do so change
into the relevant folder and then run that single file with `jest` with
`UPDATE_SNAPSHOTS` enabled:

```shell
cd postgraphile/postgraphile
UPDATE_SNAPSHOTS=1 yarn jest __tests__/path/to/test.file.graphql
```
````